src/Model.ts

```diff
@@ -674,5 +674,10 @@ function getDurationFrom(startSec: number) {
     }
     
     const endSec = (new Date().getTime()) / 1000;
-    return endSec - startSec;
+    const duration = endSec - startSec;
+    // prevent negative durations
```

Comment:
describe when negative durations can occur

and can we have a testcase? :)

Reply:
Done